packages/web-components/src/components/pricing-table/__stories__/pricing-table.stories.ts

```diff
@@ -0,0 +1,287 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, TemplateResult } from 'lit-element';
+import { number, text } from '@storybook/addon-knobs';
+import readme from './README.stories.mdx';
+import '../index';
+import 'carbon-web-components/es/components/tooltip/index';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+import { PRICING_TABLE_HEADER_CELL_TYPES } from '../defs';
+
+enum CELL_TYPES {
+  TEXT = 'text',
+  ICON = 'icon',
+  EMPTY = 'empty',
+}
+
+const renderHeaderCell = (iterator: number): TemplateResult => {
+  const cellVariations = [
+    html`
+      <dds-pricing-table-header-cell>
+        <dds-pricing-table-header-cell-headline>Variation ${iterator}</dds-pricing-table-header-cell-headline>
+        <dds-pricing-table-header-cell-caption>Starting at $X.XX per user</dds-pricing-table-header-cell-caption>
+        <dds-pricing-table-header-cell-tag href="https://www.carbondesignsystem.com/all-about-carbon/what-is-carbon/"
+          >Merchandising offer</dds-pricing-table-header-cell-tag
+        >
+        <dds-pricing-table-header-cell-description>
+          Lorem ipsum dolor sit amet consectetur.
+          <bx-unordered-list>
+            <bx-list-item>Lorem ipsum dolor</bx-list-item>
+            <bx-list-item>sit amet</bx-list-item>
+            <bx-list-item>consectetur retention adispiscing elit sed do eiusm Eiusmod tempor</bx-list-item>
+          </bx-unordered-list>
+        </dds-pricing-table-header-cell-description>
+        <dds-pricing-table-header-cell-cta href="https://www.carbondesignsystem.com/all-about-carbon/what-is-carbon/"
```

Comment:
I know that @jwitkowski79's design spec doesn't include the CTA icon, however, since this component is extending from `DDSButtonCTA` and having seen the node structure of the CTA button in AEM, we'll need to add `cta-type` which will render different SVGs depending on the selection (see - https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/?path=/story/components-cta--button). I think in this storybook preview, we can just add `cta-type="local"`.

Reply:
Thank you for clarifying! I added that attribute to `<dds-pricing-table-header-cell-cta>` elements in the documentation and each of the examples.